alkira/resource_alkira_billing_tag.go

```diff
@@ -51,6 +54,17 @@ func resourceBillingTagRead(d *schema.ResourceData, meta interface{}) error {
 }
 
 func resourceBillingTagUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*alkira.AlkiraClient)
+
+	log.Printf("[INFO] Billing Tag Updating")
+	err := client.UpdateBillingTag(d.Get("tag_id").(int), d.Get("name").(string))
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+
 	return resourceBillingTagRead(d, meta)
```

Comment:
This becomes unreachable code. Should not have compiled with this.

Reply:
compiling works fine. it returns the same actually, but yep, forgot...